geoips/geoips_utils.py

```diff
@@ -361,6 +366,9 @@ def replace_geoips_paths(fname, replace_paths=None, base_paths=None):
     base_paths : list, default=None
         * List of PATHS dictionaries in which to find the "replace_paths" variables
         * If None, use geoips.filenames.base_paths
+    curly_braces: bool, default=False
```

Comment:
VSCode will expand environment variables if they are surrounded by {}!  Default will be the same as previously, so metadata YAML outputs won't change.

Reply:
I like this change, but I think it should be configurable at the CLI. There are times that outputting the ENV is most useful and times when outputting the absolute path is more useful. I'm going to approve with this change as-is, but created #297 to address the CLI issue.